staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/controller/controller.go

```diff
@@ -90,62 +96,25 @@ func (d *DynamicKMSEncryptionConfigContent) Run(ctx context.Context) {
 	// start worker for processing content
 	go wait.UntilWithContext(ctx, d.runWorker, time.Second)
```

Comment:
I could not find a reliable way to avoid leaking go routine with _sync.WaitGroup_. Reverting this to the original implementation as it also works with the Controller's new unit tests.